RF24Network.cpp

```diff
@@ -163,17 +163,20 @@ uint8_t RF24Network::update(void)
         if (frame_size < sizeof(RF24NetworkHeader) || !is_valid_address(header->to_node) || !is_valid_address(header->from_node)) {
             continue;
         }
-
-        IF_SERIAL_DEBUG(printf_P(PSTR("%u: MAC Received " PRIPSTR
+        IF_SERIAL_DEBUG(printf_P(PSTR("%u: "), millis()));
+        IF_SERIAL_DEBUG(printf_P(PSTR("MAC Received " PRIPSTR
                                       "\n\r"),
-                                 millis(), header->toString()));
+                                 header->toString()));
 #if defined(RF24_LINUX)
         if (frame_size) {
-            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("%u: FRG Rcv frame size %i\n"), millis(), frame_size););
-            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("%u: FRG Rcv frame "), millis()); const char* charPtr = reinterpret_cast<const char*>(frame_buffer); for (uint16_t i = 0; i < frame_size; i++) { printf_P(PSTR("%02X "), charPtr[i]); }; printf_P(PSTR("\n\r")));
+            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("%u: "), millis()));
+            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("FRG Rcv frame size %i\n"), frame_size););
+            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("%u: "), millis()));
+            IF_SERIAL_DEBUG_FRAGMENTATION_L2(printf_P(PSTR("FRG Rcv frame "); const char* charPtr = reinterpret_cast<const char*>(frame_buffer); for (uint16_t i = 0; i < frame_size; i++) { printf_P(PSTR("%02X "), charPtr[i]); }; printf_P(PSTR("\n\r")));
```

Comment:
I think you're missing a semi-colon after one of `printf_P()` statements.

This line is where everything goes wrong in the Linux CI build. But that doesn't mean it it's the only place that needs a semi-colon after `printf_P()`.